libpod/runtime_img.go

```diff
@@ -295,27 +294,19 @@ func (r *Runtime) LoadImage(ctx context.Context, name, inputFile string, writer
 			return dockerarchive.ParseReference(inputFile)
 		},
 		func() (types.ImageReference, error) {
-			return ociarchive.NewReference(inputFile, name) // name may be ""
-		},
-		func() (types.ImageReference, error) {
-			// prepend "localhost/" to support local image saved with this semantics
-			if !strings.Contains(name, "/") {
-				return ociarchive.NewReference(inputFile, fmt.Sprintf("%s/%s", image.DefaultLocalRegistry, name))
+			if name != "" {
+				return nil, errors.Errorf("oci format does not support additional name %s", name)
 			}
-			return nil, nil
+			return ociarchive.NewReference(inputFile, name)
```

Comment:
```suggestion
			return ociarchive.NewReference(inputFile, "")
```

Reply:
I just think it makes it a bit more definitive, first read through I was wondering why you checked for not an empty  string then passed the var.  YMMV